src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -8,12 +8,14 @@
 
   &__search-input {
     text-align: left;
-    font-size: 0.9em;
+    font-size: 12px;
+    height: 32px;
```

Comment:
Maybe a variable at the top of the file with the name related to the use.

Reply:
fixed 433dfe8